database-mongodb/index.js

```diff
@@ -1,10 +1,6 @@
 const mongoose = require('mongoose');
-const database = mongoose.connect('mongodb://localhost/seed');
-const db = mongoose.connection; 
 
-db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', function() {
-  console.log("connected");
-});
+const database = mongoose.connect('mongodb://localhost/campsites');
```

Comment:
Not sure if you get a DeprecationWarning about the current URL string parser when you run this, but if so, you can add { useNewUrlParser: true } to your connection to avoid the warning.